Now (after the update) I get two build errors

@kitsonk 
While the update described in https://github.com/dojo/widget-core/issues/626 _should_ work now, I get the following errors when `$` `dojo build -w`

[... update :] 
seeAlso https://github.com/darul75/web-react/issues/12#issuecomment-234809701 ff.
after adding
```
"webpack": "^3.4.1",
"webpack-dev-server": "^2.6.1",
``` 
to my _project's_ `package.json` most errors are gone, just

```
ERROR in /Users/sebi/Desktop/redaktorTS2/src/webcomponents/semantic/node_modules/@dojo/shim/Promise.d.ts
(14,40): error TS2693: 'Promise' only refers to a type, but is being used as a value here.

ERROR in ./~/@dojo/core/text.js
Module not found: Error: Can't resolve 'fs' in '/Users/sebi/Desktop/redaktorTS2/src/webcomponents/semantic/node_modules/@dojo/core'
 @ ./~/@dojo/core/text.js 34:65-78
 @ ./~/@dojo/core/main.js
 @ ./src/lib/SUI.ts
 @ ./src/widgets/Container/Container.ts
 @ ./src/App.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client? ./src/main.css ./src/main.ts
```

will figure out ...

--- a/src/webpack.config.ts
+++ b/src/webpack.config.ts
@@ -118,6 +118,12 @@ function webpackConfig(args: Partial<BuildArgs>) {
 				})
 			};
 		}),
+		node: {
+			dgram: 'empty',
+			net: 'empty',
+			tls: 'empty',
+			fs: 'empty'
+		},
 		plugins: [
 			new AutoRequireWebpackPlugin(/src\/main/),
 			new webpack.BannerPlugin(readFileSync(require.resolve(`${packagePath}/banner.md`), 'utf8')),

--- a/src/webpack.d.ts
+++ b/src/webpack.d.ts
@@ -99,13 +99,13 @@ declare module 'webpack/lib/webpack' {
 		}
 
 		interface Node {
-			console?: boolean;
+			console?: boolean | 'mock';
 			global?: boolean;
 			process?: boolean;
-			__filename?: boolean | 'empty' | 'mock' | undefined;
-			__dirname?: boolean | 'empty' | 'mock' | undefined;
-			Buffer?: boolean;
-			setImmediate?: boolean;
+			__filename?: boolean | 'mock';
+			__dirname?: boolean | 'mock';
+			Buffer?: boolean | 'mock';
+			setImmediate?: boolean | 'mock' | 'empty';
 			[moduleName: string]: boolean | 'empty' | 'mock' | undefined;
 		}
 
@@ -113,7 +113,7 @@ declare module 'webpack/lib/webpack' {
 			entry: string | string[] | { [key: string]: string | string[] };
 			output: Output;
 			module: Module;
-			node: Node;
+			node?: Node | false;
 			resolve?: Resolve;
 			resolveLoader?: {
 				modules?: string[];

--- a/src/webpack.d.ts
+++ b/src/webpack.d.ts
@@ -98,10 +98,22 @@ declare module 'webpack/lib/webpack' {
 			test?: Condition;
 		}
 
+		interface Node {
+			console?: boolean;
+			global?: boolean;
+			process?: boolean;
+			__filename?: boolean | 'empty' | 'mock' | undefined;
+			__dirname?: boolean | 'empty' | 'mock' | undefined;
+			Buffer?: boolean;
+			setImmediate?: boolean;
+			[moduleName: string]: boolean | 'empty' | 'mock' | undefined;
+		}
+
 		interface Config {
 			entry: string | string[] | { [key: string]: string | string[] };
 			output: Output;
 			module: Module;
+			node: Node;
 			resolve?: Resolve;
 			resolveLoader?: {
 				modules?: string[];

